### PR TITLE
Add bio for GitHub Teacher

### PR DESCRIPTION
**Pull request by githubteacher**

### CODE REVIEW

**Comment by stephentolan** [comment]
looks good , no issues

**Comment by ajavier** [comment]
You miss the githubteacher's birthdate in the bio. Please, fix it accordingly.

File: githubteacher.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# GitHub Teacher
+
+* Name: Joshua Wehner
```
Comment (githubteacher) [inline githubteacher.md:3]: Clean up the formatting a little here.

Comment (sophiaxzhou) [inline githubteacher.md:3]: reviewed

File: githubteacher.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# GitHub Teacher
```
Comment (sophiaxzhou) [inline githubteacher.md:1]: Add comments for the first line

File: githubteacher.md (hunk 1 above)
Comment (BulandMalik) [inline githubteacher.md:3]: reviewed

**Comment by ottlinger** [comment]
I think you should add more information about the weather, thanks. @githubteacher

File: githubteacher.md (hunk 1 above)
Comment (ottlinger) [inline githubteacher.md:3]: Please omit this line.

**Pull request merged**